src/pipeline/rackhd/ova/FunctionTest.groovy

```diff
@@ -9,7 +9,23 @@ def keepEnv(String library_dir, boolean keep_env, int keep_minutes, String test_
     }
 }
 
-def runTest(String stack_type, String test_name, ArrayList<String> used_resources, Map ova_dict, boolean keep_env_on_failure, int keep_minutes,String ova_url=""){
+def runTest(String stack_type,
+            String test_name,
+            ArrayList<String> used_resources,
+            String sudo_creds,
+            String esxi_host,
+            String esxi_creds,
+            String datastore,
+            String ova_creds,
+            String ova_gateway,
+            String ova_internal_ip_creds,
+            Map ova_stash_dict=null,
+            String ova_url="",
+            String external_vswitch=null,
+            String ova_net_interface=null,
+            String dns_server_ip = null,
+            boolean keep_env_on_failure=false,
```

Comment:
you didn't use ```keep_env_on_failure``` and ```keep_minutes```

Reply:
sry, I comment the related code for debug.